main.py

```diff
@@ -1,6 +1,73 @@
-from flask import Flask
+import webbrowser
+from flask import Flask, redirect, url_for, session, request, jsonify
+
+from flask_oauthlib.client import OAuth
+
+
 app = Flask(__name__)
+app.config['GOOGLE_ID'] = "403847176041-1a3qahjkf5m786p54qhu5u5c2qvt0rkt.apps.googleusercontent.com"
+app.config['GOOGLE_SECRET'] = "2YyK88tfPFMDXwtm-2p8kQC9"
+app.debug = True
+app.secret_key = 'development'
+oauth = OAuth(app)
+
+Qruri = "https://api.qrserver.com/v1/create-qr-code/?size=150x150&data="
```

Comment:
why is Q in caps?

Reply:
Ohk will try  to improve my coding standard